programs/vote/src/vote_state/mod.rs

```diff
@@ -341,9 +340,8 @@ impl VoteState {
     pub fn process_slot(&mut self, slot: Slot, epoch: Epoch) {
         // Ignore votes for slots earlier than we already have votes for
         if self
-            .votes
-            .back()
-            .map_or(false, |old_vote| old_vote.slot >= slot)
```

Comment:
Reversed the condition operands for readability. function arguments should be first to be like a subject in a sentence.